README.md

```diff
@@ -66,13 +70,17 @@ Symbol                              | NET35              | NET40              |
 
 * `FEATURE_APPDOMAIN` - enables support for features that make use of an AppDomain in the host.
 * `FEATURE_ASSEMBLYBUILDER_SAVE` - enabled support for saving the dynamically generated proxy assembly.
+* `FEATURE_COM` - enables support for COM Interop.
 * `FEATURE_EMIT_CUSTOMMODIFIERS` - enables emitting optional and required custom modifiers defined on parameters including return parameters. It seems like a defect in corefx not to expose these methods because they are still implemented.
 * `FEATURE_EVENTLOG` - provides a diagnostics logger using the Windows Event Log.
 * `FEATURE_GAC` - enables support for obtaining assemblies using an assembly long form name.
 * `FEATURE_GET_REFERENCED_ASSEMBLIES` - enables code that takes advantage of System.Reflection.Assembly.GetReferencedAssemblies().
 * `FEATURE_LEGACY_REFLECTION_API` - provides a shim for .NET 3.5/4.0 that emulates the `TypeInfo` API available in .NET 4.5+ and .NET Core.
+* `FEATURE_NETCORE_CONVERTER_API` - provides shims to implement missing Converter in .NET Core.
+* `FEATURE_NETCORE_EXCEPTION_API` - provides shims to implement missing exception types in .NET Core.
 * `FEATURE_NETCORE_REFLECTION_API` - provides shims to implement missing functionality in .NET Core that has no alternatives.
 * `FEATURE_REMOTING` - supports remoting on various types including inheriting from MarshalByRefObject.
+* `FEATURE_RHINOMOCKS` - enables the RhinoMocks tests.

```

Comment:
I'd remove this symbol and use `FEATURE_COM` since that test file just has unit tests that use the old RhinoMocks COM interface (i.e. `Rhino.Mocks.CPP.Interfaces.dll`). Maybe `FEATURE_COM` should be `FEATURE_TESTS_COM`.